src/index.html

```diff
@@ -11,7 +11,185 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post">
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">Surname:</label>
+        <input
+      class="duo"
+      type="text"
+      id="surname"
+      name="info"
+      autocomplete="off"
+        >
+        <br>
+        <label for="name">Name:</label>
+        <input
+      class="duo"
+      type="text"
+      id="name"
+      name="information"
+      autocomplete="off"
+        >
+        <br>
+        <label for="age">How old are You?</label>
+        <input
+          class="duo"
+          type="number"
+          id="age"
+          name="age"
+          value="12"
+          min="1"
+          max="100"
+        >
+        <br>
+        <label for="date">Full date of birth:</label>
+        <input
+      class="duo"
+      type="date"
+      id="date"
+      name="date"
+        >
+        <br>
+        <label for="box">I accept the term of the agreement</label>
+        <input
+      type="checkbox"
+      id="box"
+      name="box"
+        >
+        <br>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label for="adress">E-mail:</label>
+        <input
+      type="email"
+      id="adress"
+      class="duo"
+      name="adress"
+      placeholder="email@example.com"
+        >
+        <br>
+        <label for="secret">Password:</label>
+        <input
+      type="password"
+      id="secret"
+      name="secret"
+      required
+      autocomplete="off"
+      minlength="8"
+      maxlength="15"
+        >
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <div class="duo">
+          <label for="cat">Do you love cats?</label>
+          <input
```

Comment:
Add a label to each radio button. It doesn't work correctly now 

https://github.com/mate-academy/layout_html-form/assets/99403198/3bdcf968-348d-4c75-99d9-d3c1d0ab22cc